packages/domains-table/src/domains-table/domains-table.tsx

```diff
@@ -363,6 +367,13 @@ export const DomainsTable = ( props: DomainsTableProps ) => {
 		Object.values< DomainData[] >( fetchedSiteDomains ).flat()
 	);
 
+	const currentUsersOwnsAllSelectedDomains = ! Array.from( selectedDomains ).some( ( selected ) =>
+		( domains || [] ).find(
+			( domain ) =>
+				`${ domain.domain }${ domain.blog_id }` === selected && ! domain.current_user_is_owner
```

Comment:
Can we extract this into a hashing function and reuse it in other places? A terrible outcome is we have inconsistencies in something so subtle after changing one of the places 😓